### ISSUE TITLE
Insanely high I/O read bytes

### ISSUE THREAD

**Issue by Osiris-Team** [ActivityWatch/activitywatch#715]
![image](https://user-images.githubusercontent.com/59899645/154819768-89b62d25-f328-4aaa-8b27-4de6abb5b73e.png)
Just found that the amount of data read by activitywatch is insanely high relative to all the other processes.
Is anyone else having this issue?
I am using this tool: https://docs.microsoft.com/en-us/sysinternals/downloads/process-explorer to check the process.

**Comment by ErikBjare** [comment]
@Osiris-Team We've seen some instances of this before, not sure what to do about it though. 

You could try running aw-server-rust instead and see if it does less IO.

**Comment by Osiris-Team** [comment]
Ok 👌

**Comment by ArthurusDent** [comment]
Looks like I'm having the same problem. 

The screenshot was taken at about 14 days uptime: 43 TB read, 1 TB written.

![ActivityWatch crazy IO - SSD - Screenshot 2023-04-28 cutout](https://user-images.githubusercontent.com/6848629/235360339-756fe8b4-af97-4fa2-84e3-c4106d894992.jpg)

Any updates or new advice on this issue? Do you need logs or should I just try running aw-server-rust?

**Comment by Osiris-Team** [comment]
@ArthurusDent Thats wild! After a few months you can buy a new drive lol.

**Comment by ArthurusDent** [comment]
I took the plunge and switched to aw-server-rust. Was much less hassle than expected. The docs explain it here: https://docs.activitywatch.net/en/latest/migrating.html 

Results after about 24 h:
![grafik](https://github.com/ActivityWatch/activitywatch/assets/6848629/901cf8d7-7fba-4611-b13b-af5bb662988a)

There might have been one small incosistency with the database migration. When I looked through the past few days I saw some data where I know that my pc was off. Other than that, I couldn't find any problems.
